codec-http/src/main/java/io/netty/handler/codec/http/HttpContentCompressor.java

```diff
@@ -32,6 +32,7 @@
     private final int compressionLevel;
     private final int windowBits;
     private final int memLevel;
+    protected final int compressThreshold;
```

Comment:
Why is this protected?

Reply:
Hi @ejona86. 

Users can implement the subclass to overwrite the useCompression method, which may access the compressThreshold.